tidb-cloud/tidb-cloud-dm-precheck-and-troubleshooting.md

```diff
@@ -87,6 +87,16 @@ The TiDB cluster storage is running low. It is recommended to [increase the TiKV
 
 Failed to connect to the source database. It is recommended to check whether the source database is started, the number of database connections has not reached the upper limit, and you can connect using the parameters specified by the job. After confirming that the source database is available, you can try to resume the job by clicking **Restart**.
 
+### Error message: "Access denied to the source '{bucket_uri}' using the access key ID '{access_key_id}' and the secret access key '{secret_access_key}'"
```

Comment:
This page is about dm. You should put this into `troubleshoot-import-access-denied-error`